src/modules/FileLocksmith/FileLocksmithUI/MainWindow.xaml

```diff
@@ -17,23 +17,34 @@
         <MicaBackdrop />
     </Window.SystemBackdrop>
     <Grid>
-
         <Grid.RowDefinitions>
-            <RowDefinition Height="32" />
+            <RowDefinition Height="Auto" />
             <RowDefinition Height="*" />
         </Grid.RowDefinitions>
-        <Grid x:Name="titleBar">
-            <StackPanel
-                Margin="16,8,8,8"
-                VerticalAlignment="Top"
-                Orientation="Horizontal">
-                <Image Source="Assets/AppList.png" />
-                <TextBlock
-                    x:Uid="AppTitleText"
-                    Margin="12,0,0,0"
-                    Style="{StaticResource CaptionTextBlockStyle}" />
-            </StackPanel>
+        <Grid x:Name="AppTitleBar"
+              Height="32"
+              ColumnSpacing="16">
+            <Grid.ColumnDefinitions>
+                <ColumnDefinition x:Name="LeftPaddingColumn" Width="0"/>
```

Comment:
No need for the `LeftPaddingColumn` / `RightDragColumn` / `RightPaddingColumn`

Reply:
Technically, you're 100% correct. But it doesn't cost anything either. It does make the code more consistent throughout different modules. And if, for any reason, UI would change in the future, the base is already there :)

Reply:
Would ideally be replaced with a `TitleBar`. This doesn't add a lot - just potential complexity or confusion?

Reply:
Focus will be mainly around shipping 8.0 in the coming weeks.. I guess after that, as it needs some thorough reviewing :)